pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -72,6 +74,12 @@ func add(mgr manager.Manager, r *ReconcileOperConfig) error {
 		return err
 	}
 
+	// watch for changes in the ovs-flows-config map
+	err = flowsconfig.WatchForConfigMap(c)
```

Comment:
super small nit: if err = flowsconfig... {
}
golang idiomatic

Reply:
Done. (I did it that way to be consistent with the style of the rest of the function)